docs/concepts/en-US/about_PSDocs_Azure_Configuration.md

```diff
@@ -48,6 +48,38 @@ configuration:
   AZURE_USE_PARAMETER_FILE_SNIPPET: false
 ```
 
+
+- [AZURE_USE_COMMAND_LINE_SNIPPET](#azure_use_command_line_snippet)
```

Comment:
You can add this up after line 20 to make it more like a table of contents.

You can also add a link in README.md

Reply:
Thanks @BernieWhite - updated.